src/ui/PrintableUri/PrintableUri.ts

```diff
@@ -1,34 +1,35 @@
-import { Component } from '../Base/Component';
-import { IComponentBindings } from '../Base/ComponentBindings';
 import { ComponentOptions } from '../Base/ComponentOptions';
 import { IQueryResult } from '../../rest/QueryResult';
 import { HighlightUtils, StringAndHoles } from '../../utils/HighlightUtils';
 import { Initialization } from '../Base/Initialization';
-import { analyticsActionCauseList } from '../Analytics/AnalyticsActionListMeta';
 import { Utils } from '../../utils/Utils';
 import { $$ } from '../../utils/Dom';
 import { exportGlobally } from '../../GlobalExports';
-
 import 'styling/_PrintableUri';
+import { ResultLink } from '../ResultLink/ResultLink';
+import { IResultLinkOptions } from '../ResultLink/ResultLinkOptions';
+import { IResultsComponentBindings } from '../Base/ResultsComponentBindings';
+import { StreamHighlightUtils } from '../../utils/StreamHighlightUtils';
+import * as _ from 'underscore';
+
 
-export interface IPrintableUriOptions {
+export interface IPrintableUriOptions extends IResultLinkOptions {
 }
 
 /**
  * The PrintableUri component displays the URI, or path, to access a result.
  *
  * This component is a result template component (see [Result Templates](https://developers.coveo.com/x/aIGfAQ)).
```

Comment:
We should update the JS DOC to explain that it inherits from a `ResultLink` and that it supports all options.